lang/en.json

```diff
@@ -480,6 +480,9 @@
       "VisibleElementTextVariableName": "Visible Element Text",
       "VisibleElementUrlVariableDescription": "The value of the link URL (href) of an element that was matched when the visible element trigger was triggered.",
       "VisibleElementUrlVariableName": "Visible Element Destination URL",
+      "VisualWebsiteOptimizerTagDescription": "Visual Website Optimizer is a all-in-one A/B testing and Conversion Optimization Platform.",
+      "VisualWebsiteOptimizerTagHelp": "This tag adds the asynchronous VWO Smart Code to your website.",
+      "VisualWebsiteOptimizerTagName": "Visual Website Optimizer",
```

Comment:
Do you think the name should be translated?

Reply:
I am not sure how to supply a name with spaces without this translation. Is there a way to do it?

Reply:
Hi,

You can do it like this:

https://github.com/matomo-org/tag-manager/blob/191f6b86ea892a50eadd66a7f659440a2f1779b2/Template/Tag/TawkToTag.php#L19-L21